src/test/java/net/masterthought/cucumber/ConfigurationTest.java

```diff
@@ -457,4 +457,20 @@ public void addCustomJsFiles_addsPropertyFiles() {
         List<String> returnedJsFiles = configuration.getCustomJsFiles();
         assertThat(returnedJsFiles).containsExactly("custom-code.js");
     }
+
+    @Test
+    public void getEmbeddingDirectory_CreatesCorrectDirectoryPath() {
+
+        // given
+        File reportDirectory = new File("target");
+        Configuration configuration = new Configuration(reportDirectory, "myProject");
+        String expectedPath = new File(reportDirectory, ReportBuilder.BASE_DIRECTORY +
```

Comment:
I think you misunderstand how File works. It represents the abstract path it does not do what you write in method name above
https://docs.oracle.com/javase/8/docs/api/java/io/File.html